package.json

```diff
@@ -2,92 +2,72 @@
   "name": "cdp-node-backend-template",
   "version": "0.1.0",
   "description": "CDP Backend Template",
-  "main": ".server/index.js",
+  "main": "src/index.js",
   "type": "module",
   "engines": {
     "node": ">=22"
   },
   "scripts": {
-    "build": "npm run build:server",
-    "build:server": "NODE_ENV=production babel --delete-dir-on-start --ignore \"**/*.test.js\" --ignore \"**/__fixtures__\" --ignore \"**/test-helpers\" --copy-files --no-copy-ignored --out-dir ./.server ./src",
     "docker:dev": "NODE_ENV=development npm run server:watch",
     "dev": "npm run server:watch",
     "dev:debug": "npm run server:debug",
-    "format": "prettier --write \"src/**/*.js\" \"**/*.{js,cjs,md,json,config.js,test.js,graphql.js}\"",
-    "format:check": "prettier --check \"src/**/*.js\" \"**/*.{js,cjs,md,json,config.js,test.js,graphql.js}\"",
-    "git:pre-commit-hook": "npm run format:check && npm run lint && npm test",
+    "format": "prettier --write \"**/*.{md,json}\"",
+    "format:check": "prettier --check \"**/*.{md,json,config.js}\"",
+    "git:pre-commit-hook": "npm run format:check && npm run lint",
     "postinstall": "npm run setup:husky",
-    "lint": "run-s lint:js lint:types",
-    "lint:fix": "eslint --cache . --fix",
-    "lint:js": "eslint --cache .",
-    "lint:types": "tsc --build tsconfig.json",
+    "lint": "standard",
+    "lint:fix": "standard --fix",
```

Comment:
Defra standards are to use StandardJs for linting.

https://defra.github.io/software-development-standards/standards/javascript_standards/#use-standard-js-to-lint-your-code

There is some debate over the future of StandardJs but engagement with them so far has not led to any reason to move away yet.

The reason StandardJs was chosen was that it's fairly straightforward and the out of the box rules "just work for us".  So no need for lots of additional custom opinionated setup.

Have therefore removed eslint and associated opinions.

Similarly have removed js files from prettier as StandardJs conflicts with Prettier.

Reply:
Not sure if your aware but the eslint config thats used in standard is the same eslint config we use here. And that works with prettier and doesn't conflict. Also the other eslint rules are all best practice for node and catch quite a bit.

Shame to see this go as a lot of work was put in to make it work.

We have a ticket on the backlog around [Eslint v8 has been marked as deprecated - how difficult is it to update to v9? - 488665](https://dev.azure.com/defragovuk/DEFRA-CDP/_workitems/edit/488665) which also mentions having a look at neostandard. Not sure how you feel about this?

The standard repo feels like its having some internal team turmoil